PokeTraining/forms.py

```diff
@@ -13,7 +11,7 @@ class Meta:
 
     def save(self, commit=True):
         pokemon = super(PokemonForm, self).save(commit=False)
-        entered_pkmn = pokepy.V2Client().get_pokemon(self.cleaned_data['pkmn_name'])
+        entered_pkmn = pb.pokemon(self.cleaned_data['pkmn_name'])
```

Comment:
Testing (and perhaps refactoring) will be needed. Instead of making an API call upon object creation, create the object, and then leave the API call for a task that can be done afterwards.

Reply:
Issue #4 has been created for this.